import-scripts/subset-impact-data.sh

```diff
@@ -55,6 +55,23 @@ if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
 fi
 echo -e "\tPORTAL_SCRIPTS_DIRECTORY="$PORTAL_SCRIPTS_DIRECTORY
 
+# check for presence of new filename pattens (not yet supported) and fail if present
+
+$PYTHON_BINARY $PORTAL_SCRIPTS_DIRECTORY/study_directory_uses_updated_filenames.py $INPUT_DIRECTORY $STUDY_ID
+if [ $? -eq 0 ] ; then
+    echo "unsupported filenames present - skipping subset" >&2
+    exit 1
+fi
+
+PYTHON_FILENAME_CHECK_CALL="import sys
```

Comment:
is this duplicate of the section above? if not, we should have the call be similar to the one before, or if it is remove this

Reply:
I thought I did remove this .... yes - no longer needed